service/worker/deployment/deployment_workflow.go

```diff
@@ -51,31 +46,28 @@ type (
 	}
 )
 
-var (
```

Comment:
Removing this for now since I haven't drafted activities yet. Makes sense to include this as and when I do